packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -137,9 +137,9 @@ const TreeItem = React.forwardRef(function TreeItem(props, ref) {
     let flag = false;
     const key = event.key;
 
-    const printableCharacter = () => {
+    const printableCharacter = e => {
```

Comment:
```suggestion
    const printableCharacter = event => {
```

Reply:
shadowed by the event above it :(

Reply:
I was being lazy, but I already have a fix.